src/data_sets.h

```diff
@@ -1456,6 +1456,8 @@ enum eSystemCommand
     SYS_CMD_GPX_ENABLE_GNSS1_PASS_THROUGH               = 33,           // (uint32 inv: 4294967262)
     SYS_CMD_GPX_ENABLE_GNSS2_PASS_THROUGH               = 34,           // (uint32 inv: 4294967261)
     SYS_CMD_GPX_ENABLE_SERIAL_BRIDGE_CUR_PORT_LOOPBACK  = 35,           // (uint32 inv: 4294967260) // Enables serial bridge on IMX to GPX and loopback on GPX.
+    SYS_CMD_GPX_REQUEST_GNSS1_HARD_RESET                = 36,           // (uint32 inv: 4294967259)
+    SYS_CMD_GPX_REQUEST_GNSS2_HARD_RESET                = 37,           // (uint32 inv: 4294967258)
```

Comment:
Looks good.  But why do you call it a request?  Is there any question that the GNSS reset command might get rejected or significantly delayed?

SYS_CMD_GPX_HARD_RESET_GNSS1
SYS_CMD_GPX_HARD_RESET_GNSS2